lib/apiservers/service/restapi/handlers/encode/networking.go

```diff
@@ -94,20 +94,28 @@ func AsNetwork(network *executor.NetworkEndpoint) *models.Network {
 	return m
 }
 
-func AsImageFetchProxy(sessionConfig *executor.SessionConfig, http, https string) *models.VCHRegistryImageFetchProxy {
+func AsImageFetchProxy(sessionConfig *executor.SessionConfig, http, https, nproxy string) *models.VCHRegistryImageFetchProxy {
 	var httpProxy, httpsProxy strfmt.URI
+	var nProxy []strfmt.URI
 	for _, env := range sessionConfig.Cmd.Env {
 		if strings.HasPrefix(env, http+"=") {
 			httpProxy = strfmt.URI(strings.SplitN(env, "=", 2)[1])
 		}
 		if strings.HasPrefix(env, https+"=") {
 			httpsProxy = strfmt.URI(strings.SplitN(env, "=", 2)[1])
 		}
+		if strings.HasPrefix(env, nproxy+"=") {
+			nProxyStrs := strings.Split(strings.SplitN(env, "=", 2)[1], ",")
+			nProxy = make([]strfmt.URI, len(nProxyStrs))
+			for i := range nProxy {
+				nProxy[i] = strfmt.URI(strings.TrimSpace(nProxyStrs[i]))
```

Comment:
To be consistent with the other functions in this package (where each function converts one object), this should probably be extracted into its own function (e.g., `asProxyExclusionList`).